docs/references.md

```diff
@@ -17,47 +17,137 @@ edition of the referenced document (including any amendments) applies.
 The MITRE Corporation,
 [https://cpe.mitre.org/files/cpe-specification_2.2.pdf](https://cpe.mitre.org/files/cpe-specification_2.2.pdf)
 
-NIST IR 7695, *Common Platform Enumeration: Naming Specification Version 2.3*,
+*Common Platform Enumeration: Naming Specification Version 2.3*, NIST IR 7695,
```

Comment:
```suggestion
*Common Platform Enumeration(CPE): Naming Specification Version 2.3*, NIST IR 7695,
```

Reply:
From @kestewart suggestion, there should be a space before `(CPE)`.

Reply:
I can see it fixed now